diff --git a/node_modules/npm-solarsystem/index.js b/node_modules/npm-solarsystem/index.js
index cc4ef86..cfac192 100644
--- a/node_modules/npm-solarsystem/index.js
+++ b/node_modules/npm-solarsystem/index.js
@@ -53,7 +53,7 @@ module.exports.getEarth = function Earth() {
 
 module.exports.getMars = function Mars() {
     const mars = {
-        image: 'https://www.nasa.gov/sites/default/files/thumbnails/image/edu_what_is_mars.jpg',
+        image: 'https://upload.wikimedia.org/wikipedia/commons/thumb/0/0c/Mars_-_August_30_2021_-_Flickr_-_Kevin_M._Gill.png/960px-Mars_-_August_30_2021_-_Flickr_-_Kevin_M._Gill.png',
         distanceFromSun: '145.925 million miles',
         yearLength: '687 Earth Days',
         description: 'The fourth planet from the Sun, Mars is a dusty, cold, desert world with a very thin atmosphere. This dynamic planet has seasons, polar ice caps and weather and canyons and extinct volacanoes, evidence of an even more active past.',
@@ -68,7 +68,7 @@ module.exports.getMars = function Mars() {
 
 module.exports.getJupiter = function Jupiter() {
     const jupiter = {
-        image:'https://www.nasa.gov/sites/default/files/thumbnails/image/hs-2016-24-a-print-new.jpg',
+        image:'https://upload.wikimedia.org/wikipedia/commons/e/e2/Jupiter_OPAL_2024.png',
         distanceFromSun: '485.316 million miles',
         yearLength: '4333 Earth Days',
         description: 'Jupiter is the fifth planet from our Sun and is, by far, the largest planet in the solar system – more than twice as massive as all the other planets combined. Jupiters stripes and swirls are actually cold, windy clouds of ammonia and water, floating in an atmosphere of hydrogen and helium. Jupiter’s iconic Great Red Spot is a giant storm bigger than Earth that has raged for hundreds of years. Jupiter is surrounded by dozens of moons. Jupiter also has several rings, but unlike the famous rings of Saturn, Jupiter’s rings are very faint and made of dust, not ice.',
@@ -98,7 +98,7 @@ module.exports.getSaturn = function Saturn() {
 
 module.exports.getUranus = function Uranus() {
     const uranus = {
-        image: 'https://images.ctfassets.net/cnu0m8re1exe/7hQBr1Got3iOOxCn4yJqSy/8ac59ceb4d08d0474a2688ac976b1693/neptune1-1024x727.jpg?w=650&h=433&fit=fill',
+        image: 'https://upload.wikimedia.org/wikipedia/commons/6/69/Uranus_Voyager2_color_calibrated.png',
         distanceFromSun: '1.833 billion miles',
         yearLength: '30687 Earth Days',
         description: 'The seventh planet from the Sun with the third largest diameter in our solar system, Uranus is very cold and windy. The ice giant is surrounded by 13 faint rings and 27 small moons as it rotates at a nearly 90-degree angle from the plane of its orbit. This unique tilt makes Uranus appear to spin on its side, orbiting the Sun like a rolling ball. ',
@@ -128,7 +128,7 @@ module.exports.getNeptune = function Neptune() {
 
 module.exports.getPluto = function Pluto() {
     const pluto = {
-        image: 'https://spaceplace.nasa.gov/review/ice-dwarf/pluto_new_horizons.en.png',
+        image: 'https://upload.wikimedia.org/wikipedia/commons/c/ca/Pluto_in_True_Color_-_High-Res.png',
         distanceFromSun: '3.157 billion miles',
         yearLength: '90530 Earth Days',
         description: 'Dwarf planet. Pluto is a complex and mysterious world with mountains, valleys, plains, craters, and maybe glaciers. Discovered in 1930, Pluto was long considered our solar systems ninth planet. But after the discovery of similar intriguing worlds deeper in the distant Kuiper Belt, icy Pluto was reclassified as a dwarf planet. Pluto is orbited by five known moons, the largest of which is Charon. Charon is about half the size of Pluto itself, making it the largest satellite relative to the planet it orbits in our solar system. Pluto and Charon are often referred to as a "double planet."',
